examples/nfs/README.md

```diff
@@ -1,38 +1,27 @@
 <!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
 
-<!-- BEGIN STRIP_FOR_RELEASE -->
 
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
+<!-- END MUNGE: UNVERSIONED_WARNING -->
 
-<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+# Outline
```

Comment:
We are going to need to be clear that this example only works on clusters running on GCE